packages/elpi-option-legacy-parser/elpi-option-legacy-parser.1/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "2.0"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Enrico Tassi" ]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/LPCIC/elpi"
+doc: "https://LPCIC.github.io/elpi/"
+dev-repo: "git+https://github.com/LPCIC/elpi.git"
+bug-reports: "https://github.com/LPCIC/elpi/issues"
+
+depends: [
```

Comment:
It might be worth doing:
```suggestion
depends: [
  "elpi" {post}
```
to avoid issues later one for users who wouldn’t understand why installing elpi-option-legacy-parser alone does not instal elpi. It’s also annoying if a user needs to depopts against this package (the same issue appeared with the ctypes-foreign package)

Reply:
Addressed in https://github.com/ocaml/opam-repository/pull/21226/files.